src/sdgio-edit.owl

```diff
@@ -2766,6 +2778,101 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020241> "2019-04-29T16:26:25Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020241> <https://clarisa.cgiar.org/api/srf-idos/4>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020241> "increased productivity"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020241>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020242> (improved diets for poor and vulnerable people)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020242> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020242> "2019-04-29T16:27:06Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020242> <https://clarisa.cgiar.org/api/srf-idos/5>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020242> "improved diets for poor and vulnerable people"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020242>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020243> (improved food security)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020243> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020243> "2019-04-29T16:27:28Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020243> <https://clarisa.cgiar.org/api/srf-idos/6>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020243> "improved food security"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020243>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020244> (improved human and animal health through better agricultural practices)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020244> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020244> "2019-04-29T16:28:08Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020244> <https://clarisa.cgiar.org/api/srf-idos/7>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020244> "improved human and animal health through better agricultural practices"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020244>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020245> (natural capital enhanced and protected, especially from climate change)
```

Comment:
As with "improved", "enhanced" is a slipper idea. We should get information about what that means too and add it as a comment on this instance (in a separate PR)

Reply:
I added issue #148.